src/styles/generic/_layout.scss

```diff
@@ -69,63 +69,74 @@ header {
 }
 
 .function {
-  flex: 10;
-  margin: 20px 25px;
+    flex: 10;
+    margin: 20px 25px;
 }
 
 .timeline {
-  flex: 0.65;
-  &:after {
-    content: "";
-    border-left: 1px solid #888;
-    height: 87%;
-    display: block;
-    margin-top: 34px;
-    margin-left: 16px;
-    position: absolute;
+    flex: 0.65;
+    &:after {
+      content: "";
+      border-left: 1px solid #888;
+      height: 87%;
+      display: block;
+      margin-top: 34px;
+      margin-left: 16px;
+      position: absolute;
   }
 }
 
+input, textarea {
+  border: none;
+  background: rgba(0, 0, 0, 0);
+  cursor: pointer;
+  color: #fff;
+  text-transform: none;
+}
+
 //Header for CV template
 .headera4 {
-  height: 350px;
-  background:linear-gradient(0deg,rgba(0,0,0,0.7),rgba(0,0,0,0.7)),url(https://myob-gm-digital-production-blog-assets.s3.amazonaws.com/uploads/2018/11/Death-of-open-plan-offices-header.jpg);
-  padding: 100px 25px;
-  h1 {
-    color: #fff;
-    font-size: 36px;
-    text-transform: none;
-  }
-  p {
-    color: #fff;
-  }
-  input, textarea {
-    border: none;
-    background: rgba(0, 0, 0, 0);
-    cursor: pointer;
-    color: #fff;
-    text-transform: none;
-  }
+    height: 350px;
+    background:linear-gradient(0deg,rgba(0,0,0,0.7),rgba(0,0,0,0.7)),url(https://myob-gm-digital-production-blog-assets.s3.amazonaws.com/uploads/2018/11/Death-of-open-plan-offices-header.jpg);
+    padding: 100px 25px;
+    h1 {
+        color: #fff;
+        font-size: 36px;
+        text-transform: none;
+    }
+    p {
+        color: #fff;
+    }
+    input, textarea {
+      border: none;
+      background: rgba(0, 0, 0, 0);
+      cursor: pointer;
+      color: #fff;
+      text-transform: none;
+    }
   textarea {
     width: 950px;
     height: 100px;
     resize: none;
   }
-}
+  }
 
 .editable {
   cursor: pointer;
   &:hover {
-    box-shadow: 0px 0px 15px #fff;
+    //box-shadow: 0px 0px 15px #fff;
     //margin: 0px 10px 0px 10px;
+    border: 1px solid #cecece;
+    position: relative;
     border-radius: 5px;
+    background-color: #363636;
     display: inline-block
   }
 }
 
 #description {
-  font-size: 12px;
-}
+   font-size: 12px;
```

Comment:
extra space?

Reply:
hier kan je een Variablen gebruik en andere 12px waardes refactoren. main-font-size is nu 14 dus wellicht kan je die iets als secondary-font-size noemen